app/views/flexible_page/flexible_sections/_rich_content.html.erb

```diff
@@ -0,0 +1,15 @@
+<nav class="govuk-grid-column-one-third">
+  <% if flexible_section.image %>
+    <img alt="<%= flexible_section.image["alt"] %>" loading="lazy" class="govuk-!-width-full" src="<%= flexible_section.image["src"] %>">
+  <% end %>
+
+  <%= render "govuk_publishing_components/components/contents_list", {
+    contents: ContentsListPresenter.new(flexible_section.contents_list).to_component,
+  } %>
+</nav>
+
+<div class="govuk-grid-column-two-thirds">
+  <%= render "govuk_publishing_components/components/govspeak", {} do %>
+    <%= flexible_section.govspeak.html_safe %>
```

Comment:
I think initially when I wrote this it wouldn't display the govspeak properly but adding `.html_safe` fixed it, just want to check that this is the best solution for this problem.

Reply:
Yes, we want this here. The govspeak block is HTML, putting `.html_safe` in the view here is a good way to signal "we know this is HTML we want to render, and we trust it".